src/components/controllers/tutorial/style.module.css

```diff
@@ -393,10 +390,5 @@
 }
 
 .output {
-	color: #444;
 	height: 100%;
-
-	h1 {
-		color: #444;
-	}
```

Comment:
Should be entirely unused, as we load the iframe as a child of `.output` here. Spotted when playing around with the colors.